release_notes.md

```diff
@@ -7,4 +7,4 @@
 
 **Release sprint:** Sprint 113
 [ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+113%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+113%22+label%3Afeature+is%3Aclosed) ]
-- Update PowerShell Worker to 3.0.1272 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.1272)
\ No newline at end of file
+- Update PowerShell Worker to 3.0.1475 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.1475)
```

Comment:
Was this updated twice since it was last release, or were the notes not reset?

Reply:
@fabiocav -- the PowerShell 7 worker has been updated twice for V3 since the last release.